test/helpers/index.ts

```diff
@@ -17,26 +20,24 @@ const testAmounts: string[] = [
     "669", "555",
 ];
 
+// Completely clean privkey with low balances.
```

Comment:
This needs to use a hardhat provider for tests. Mocking out non-networking tests w/ a private key like this is fine, but this is too flaky for use here

Reply:
See https://github.com/synapsecns/sdk/pull/63#issuecomment-1053601134